### PR TITLE
Us002 view job spec

### PR DESCRIPTION
**Pull request by leighmcclurg**

### CODE REVIEW

File: views/job-specification.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<main class="container mt-4">
+    <section class="col d-flex justify-content-center mt-4">
+        {% if role %}
+       
+        <div class="card text-center" style="width:75%">
+            <div class="card-body">
+              <h3 class="card-title"> {{ role.jobProfileTitle }}</h3>
+            </div>
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item"> Capability: {{ role.capability }} </li>
```
Comment (glenn-hamilton) [inline views/job-specification.html]: Could you make each of the "headings" bold? For example <strong> Capability:</strong>

File: views/job-specification.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{% extends "layout.html" %} 
+ {% set page_title = "Job Specification" %} {% block main
+    %}
+<main class="container mt-4">
+    <section class="col d-flex justify-content-center mt-4" id="job-specification">
+        {% if role %}
+       
+        <div class="card text-center card-border" style="width:75%">
```
Comment (glenn-hamilton) [inline views/job-specification.html]: Inline CSS isn't a great idea `style="width:75%"`

File: .pa11yci.conf.js [hunk 3]
```diff
@@ -7,7 +7,11 @@ var defaults = {
     },
 }
 
-var urls = ['${PA11Y_TEST}']
+var urls = [
```
Comment (glenn-hamilton) [inline .pa11yci.conf.js:10]: Can you run the accessibility tests locally just to make sure it all passes

File: controller/baseController.ts [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 module.exports = function (app: Application) {
-    app.get('/helloworld', async (req: Request, res: Response) => {
-        res.render('helloWorld')
-    })
-
     app.get('/', async (req: Request, res: Response) => {
         res.render('index')
     })
+
+    app.get('/login', async (req: Request, res: Response) => {
```
Comment (glenn-hamilton) [inline controller/baseController.ts]: How did this get in here?

**Review by glenn-hamilton — DISMISSED** [review]

**Review by MorganMcDonald — DISMISSED** [review]

**Review by MorganMcDonald — APPROVED** [review]

**Review by glenn-hamilton — APPROVED** [review]

**Pull request merged**